core/src/display_object.rs

```diff
@@ -312,11 +313,11 @@ impl<'gc> DisplayObjectBase<'gc> {
         self.name = name;
     }
 
-    fn filters(&self) -> Avm2ArrayStorage<'gc> {
-        Avm2ArrayStorage::from_storage(self.filters.iter().collect())
+    fn filters(&self) -> Vec<Filter> {
+        self.filters.clone()
```

Comment:
Can we avoid the `.clone()` by returning a `&[Filter]`?

Reply:
Not trivially, everything in DisplayObject returns owned types because of the lifetime complexities involved in returning a ref to a gc'd value

Reply:
Yeah I remember something like that. I think nonetheless a `Ref<'gc, Vec<Filter>>` or something can be returned, but let's keep this as-is.